plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -27,11 +27,9 @@ class WC_Helper_Admin {
 	 * @return void
 	 */
 	public static function load() {
-		global $pagenow;
```

Comment:
It's not possible to use $pagenow global here for network activated plugin.

Reply:
Since we are validating the page and the path there is no need to check $pagenow.

Reply:
One thing I noticed by removing pagenow check, the block can get loaded on unauthenticated ajax request , for example:

```
curl  'https://example.com/wp-admin/admin-ajax.php?page=wc-admin&path=/extensions'
```

Not a biggie as it just register a callback in `woocommerce_admin_shared_settings` hook — which is not fire anyway.

Reply:
Yes it's not going to be an issue, I've tested with a site having the build from this branch: https://generously-preferred.jurassic.ninja/site1/wp-admin/admin-ajax.php?page=wc-admin&path=/extensions